form-sharing/2-getSheetsList.js

```diff
@@ -0,0 +1,14 @@
+getValues(
+  '1s7K3kxzm5AlpwiALattyc7D9_aIyqWmo2ubcQIUlqlY',
+  'sheetsList of kobo forms!A:K'
+);
+fn(state => {
+  const koboFormsData = state.references[1];
+  const googleSheetsData = state.data.values.map(row => row[0]);
+
+  state.data = koboFormsData.filter(
```

Comment:
I think you need the results from this step for other down stream job, `updateSheetLists` and `updateAsanaTask`. So i would probably put this data in state with a different key. Eg `state.filteredKoboFormsData` or `state.filteredKoboForms`